src/W/InputTextArea.elm

```diff
@@ -165,6 +165,8 @@ view attrs_ props =
         (attrs.htmlAttributes
             ++ [ WH.maybeAttr HA.id attrs.id
                , WH.attrIf (not attrs.unstyled) HA.class "ew ew-input ew-focusable"
+               , WH.attrIf (not attrs.unstyled) HA.class attrs.class
```

Comment:
We don't need to use the unstyled check here! :) We can just pass in the class directly.